src/data_storage/dynamo_db/data_source.py

```diff
@@ -7,43 +6,33 @@
 
 from common.entities.schemas import Node, Story, StoryList
 from conf import settings
+from data_storage.data_source import DataDriver
 from utilities.exceptions import DynamoDBError
 
 
-def _get_connection():
-    config = Config(retries={"max_attempts": 1, "mode": "standard"})
-    if settings.LOCAL:
-        conn = boto3.resource(
-            "dynamodb",
-            endpoint_url=settings.DYNAMODB_URL,
-            config=config,
-            region_name=settings.REGION,
-        )
-    else:
-        conn = boto3.resource(
-            "dynamodb",
-            config=config,
-        )
-    return conn
-
-
-class DataDriver(abc.ABC):
-    @abc.abstractmethod
-    def get_story(self, story_id: str):
-        """ "Get story from different source by story_id"""
-
-    @abc.abstractmethod
-    def get_node(self, story_id: str, uri: str):
-        """Get story node from source by story_id and nodes URI"""
-
-    @abc.abstractmethod
-    def get_story_list(self):
-        """Get list of stories"""
-
-
 class DynamoDBDriver(DataDriver):
-    def __init__(self, connection):
-        self.connection = connection.Table(settings.STORY_TABLE_NAME)
+    def __init__(self, connection=None):
+        if connection is None:
+            self.connection = self._get_connection()
+        else:
+            self.connection = connection
+        self.connection = self.connection.Table(settings.STORY_TABLE_NAME_DYNAMODB)
+
+    def _get_connection(self):
+        config = Config(retries={"max_attempts": 1, "mode": "standard"})
+        if settings.LOCAL:
+            conn = boto3.resource(
+                "dynamodb",
+                endpoint_url=settings.DYNAMODB_URL,
+                config=config,
+                region_name=settings.REGION,
+            )
+        else:
+            conn = boto3.resource(
+                "dynamodb",
+                config=config,
+            )
+        return conn
```

Comment:
I'd split Driver logic and Driver initialisation

Reply:
Need to remove `get_connection` from class and make it function?

Reply:
Yes, it might add initialising function for our drivers. 
```
def get_data_source() -> DataDriver:
    data_source = DATA_SOURCES.get(settings.data_sourse)
    if data_source is None:
        raise ValueError("Unknown driver name")
    driver_cls, settings_cls, initialiser = data_source
    data_source_settings = settings_cls()
    kwargs = get_kwargs(data_source_settings)
    driver = driver_cls(**kwargs)
    return driver
```